source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,48 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-02-24
+
+### Highlights
+
+#### Custom Statuses
+ - Users can set a custom status by using the custom status dropdown in the status menu.
+
+#### Support Packet Generation
+ - Allows a System Admin to download a support packet which provides helpful information to our internal support team.
+
+### Improvements
+
+#### User Interface (UI)
+ - Removed the 5-page limit on previewing PDFs.
+ - Added "files" as a reserved team name.
+ - Searching for a channel url_name now shows the channel.
+ - Users are now provided with feedback when creating a custom category name for a channel that exceeds the character limit.
+
+#### Administration
+ - Added support for compressed export files with attachments.
+ - Server crashes due to runtime panics are now captured as a log line.
+ - Optimized Direct Message creation by fetching all users involved in a single Database call.
+ - A change in a user's ``NotifyProps`` during the user import process will not send an email notification. This is done to make it consistent with other parts of the import process where a change in user's attributes would also not send any notifications.
+ - Implemented a job to delete unused export files.
+
+### Bug Fixes
+ - Fixed an issue where demoting a user to a guest would not take effect in an environment with read replicas.
+ - Fixed an issue where creation of a bot would fail due to replica lag.
+ - Fixed an issue where ``mmctl channel move`` did not allow moving private channels.
+ - Fixed an issue where markdown tables did not wrap correctly.
+ - Fixed an issue where the search bar styling on dark themes was off on mobile web view.
```

Comment:
What does "off" refer to in this specific context? Does it indicate disabled or something else?